SchedulerJobs/local.settings.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
local.settings shouldn't be on source control

Reply:
Please check/update your .gitignore file / better yet pull the standard .gitignore off github for c#/Azure Functions.